java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -568,6 +569,29 @@ public void hdel_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void hmget_success() {
+        // setup
+        String key = "testKey";
+        String[] fields = {"testField1", "testField2"};
+        String[] args = {"testKey", "testField1", "testField2"};
+        String[] value = {"testValue1", "testValue2"};
+
+        CompletableFuture<String[]> testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn(value);
+        when(commandManager.<String[]>submitNewCommand(eq(HashMGet), eq(args), any()))
```

Comment:
look at: https://github.com/aws/glide-for-redis/pull/1016#discussion_r1499614861